tox.ini

```diff
@@ -1,20 +1,4 @@
-[tox]
-envlist = py27, py35, py36, py37 flake8
-
-[travis]
-python =
-    3.7: py37
-    3.6: py36
-    3.5: py35
-    2.7: py27
-
-[testenv:flake8]
-basepython = python
-deps = flake8
-commands = flake8 huvr_api_client
```

Comment:
removed flake8 stuffs, and don't care as much about testing against versions of python -- we're not using any special python. and realistically won't support python2